pkg/controllers/work/apply_controller_integration_test.go

```diff
@@ -563,22 +563,17 @@ var _ = Describe("Work Controller", func() {
 		})
 
 		It("Check that failed to apply manifest has the proper identification", func() {
-			broadcastName := "testfail"
-			namespace := defaultNS
-			broadcastJob := &kruisev1alpha1.BroadcastJob{
+			cloneSet := &kruisev1alpha1.CloneSet{
 				TypeMeta: metav1.TypeMeta{
-					APIVersion: kruisev1alpha1.SchemeGroupVersion.String(),
-					Kind:       "BroadcastJob",
+					APIVersion: "invalid",
```

Comment:
why is it invalid?

Reply:
It is testing "failed to apply manifest", which means a bad manifest. The line 582 block is expecting the applied condition is false.